makefiles/py.inc.mk

```diff
@@ -0,0 +1,19 @@
+# generate .py.h header files of .py files
```

Comment:
This is tricky to get right while covering all (reasonable) use cases. Both Lua and Jerryscript leave it up to the user to define and  perform this step in the application Makefile.

Reply:
> This is tricky to get right while covering all (reasonable) use cases.

Care to elaborate?